decentraland/unity-renderer
# Fix: removed ff for new skybox

## What does this PR change?

Fix #2309 
Removes feature flag for new skybox

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/ff-skybox
2. Verify that the new skybox is visible
3. Verify that the skybox settings work as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
